external/types/web.d.ts

```diff
@@ -921,4 +921,120 @@ declare namespace _pdfjs {
 
   // Constructor function
   let GenericL10n: new (lang: string) => GenericL10n;
+
+  //
+  // web/app.js
```

Comment:
The variables added here correspond to https://github.com/mozilla/pdf.js/blob/master/web/default_preferences.json, which are only ever used in the default viewer (i.e. the one found at https://mozilla.github.io/pdf.js/web/viewer.html) and nowhere else. These are not used/exposed either through the PDF.js API, nor through https://github.com/mozilla/pdf.js/blob/master/web/pdf_viewer.component.js. Hence the default viewer, i.e. file web/app.js, shouldn't be part of any "public" API.

Reply:
I just undid this merge and squashed again, so we are back where we were before the pull request.